frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -472,12 +497,96 @@ export function PageHeaderCustom(): JSX.Element {
+                                <MakeDecisionModal experimentId={experimentId} />
+                            </>
+                        )}
                 </>
             }
         />
     )
 }
 
+export function MakeDecisionModal({ experimentId }: { experimentId: Experiment['id'] }): JSX.Element {
+    const { experiment, sortedWinProbabilities, isMakeDecisionModalOpen } = useValues(experimentLogic({ experimentId }))
+    const { closeMakeDecisionModal, shipVariant } = useActions(experimentLogic({ experimentId }))
+    const { aggregationLabel } = useValues(groupsModel)
+
+    const [selectedVariantKey, setSelectedVariantKey] = useState<string | null>()
+    useEffect(() => setSelectedVariantKey(sortedWinProbabilities[0]?.key), [sortedWinProbabilities])
+
+    const aggregationTargetName =
+        experiment.filters.aggregation_group_type_index != null
+            ? aggregationLabel(experiment.filters.aggregation_group_type_index).plural
+            : 'users'
+
+    return (
+        <LemonModal
+            isOpen={isMakeDecisionModalOpen}
+            onClose={closeMakeDecisionModal}
+            width={600}
+            title="Make decision"
+            footer={
+                <div className="flex items-center gap-2">
+                    <LemonButton type="secondary" onClick={closeMakeDecisionModal}>
+                        Cancel
+                    </LemonButton>
+                    <LemonButton onClick={() => shipVariant(selectedVariantKey)} type="primary">
+                        Ship variant
+                    </LemonButton>
+                </div>
+            }
+        >
+            <div className="space-y-4">
+                <div className="text-sm">
+                    This action will roll out the selected variant to <b>100% of {aggregationTargetName}.</b>
+                </div>
+                <LemonSelect
+                    data-attr="metrics-selector"
+                    value={selectedVariantKey}
+                    onChange={(variantKey) => setSelectedVariantKey(variantKey)}
+                    options={sortedWinProbabilities.map(({ key }) => ({
+                        value: key,
+                        label: (
+                            <div className="space-x-2 inline-flex">
+                                <VariantTag experimentId={experimentId} variantKey={key} />
+                                {key === sortedWinProbabilities[0]?.key && (
+                                    <LemonTag type="success">
+                                        <b className="uppercase">Winning</b>
+                                    </LemonTag>
+                                )}
+                            </div>
+                        ),
+                    }))}
+                />
+                <LemonBanner type="info" className="mb-4">
+                    For a more precise control over your release, adjust the rollout percentage and release conditions
```

Comment:
```suggestion
                    For more precise control over your release, adjust the rollout percentage and release conditions
```
(nit: grammar) – we don't need "a" before referring to an abstract concept.